bofire/strategies/predictives/predictive.py

```diff
@@ -102,16 +103,47 @@ def predict(self, experiments: pd.DataFrame) -> pd.DataFrame:
             experiments=experiments, specs=self.input_preprocessing_specs
         )
         preds, stds = self._predict(transformed)
+        column_names = list(
+            itertools.chain(
+                *[
+                    [f"{feat.key}_pred"]
+                    if not isinstance(feat, CategoricalOutput)
+                    else [f"{feat.key}_pred_{cat}" for cat in feat.categories]
```

Comment:
This can be added. One concern is then the user cannot see the other probabilities. For example, it could be [0.51, 0.49, 0.0] as the predicted probabilities, but if they only see the highest value the information on the closeness of the first two classes is missed. I'm fine with either way, but just want to bring a scenario like this up

Reply:
@jduerholt any comment on the above? Which functionality do you prefer?